opentracing/tracer.py

```diff
@@ -21,11 +21,10 @@
 from __future__ import absolute_import
 from concurrent.futures import Future
 from .span import Span
-from .context import TraceContextSource
-from .context import TraceContextEncoder, TraceContextDecoder
+from .propagator import SpanPropagator
 
 
-class Tracer(TraceContextSource, object):
+class Tracer(SpanPropagator):
```

Comment:
I don't have strong feeling one way or another. In practice I've always implemented "propagator" in a separate class and the tracer was delegating to it, so converting to explicit containment would be one less function call. The injectors pattern is going to change this anyway, so I'm ok to leave as is.